test/unit/raptor/RaptorDepartAfterQuery.spec.ts

```diff
@@ -29,6 +29,98 @@ describe("RaptorDepartAfterQuery", () => {
     ]);
   });
 
+  it("finds route not through via simple", () => {
+    const trips = [
+      t(st("A", null, 1000),
+        st("B", 1025, 1028),
+        st("C", 1100, null)),
+      t(st("A", null, 1010),
+        st("D", 1030, 1035),
+        st("C", 1110, null))
+    ];
+
+    const raptor = RaptorQueryFactory.createDepartAfterQuery(trips, {}, {}, calendars, journeyFactory);
+    const result = raptor.plan("A", "C", new Date("2018-10-16"), 900, ["B"]);
+
+    setDefaultTrip(result);
+
+    chai.expect(result).to.deep.equal([
+      j([
+        st("A", null, 1010),
+        st("D", 1030, 1035),
+        st("C", 1110, null)
+      ])
+    ]);
+  });
+
+  it("finds route not through via", () => {
+    const trips = [
+      t(st("A", null, 1000),
+        st("B", 1025, 1028),
+        st("C", 1100, 1108),
+        st("D", 1135, null)),
+      t(st("A", null, 1000),
+        st("E", 1035, 1040),
+        st("C", 1050, null)
+        )
+    ];
+
+    const raptor = RaptorQueryFactory.createDepartAfterQuery(trips, {}, {}, calendars, journeyFactory);
+    const result = raptor.plan("A", "D", new Date("2018-10-16"), 900, ["B"]);
+
+    setDefaultTrip(result);
+
+    chai.expect(result).to.deep.equal([
+      j([
+        st("A", null, 1000),
+        st("E", 1035, 1040),
+        st("C", 1050, null)
+      ],
+      [
+        st("C", 1100, 1108),
+        st("D", 1135, null)
+      ])
+    ]);
+  });
+
+  it("finds correct not via ignoring transfers", () => {
+      const trips = [
+          t(st("A", null, 1000),
+              st("B", 1025, 1028),
+              st("C", 1100, 1108),
+              st("D", 1135, null)),
+          t(st("A", null, 1000),
+              st("E", 1018, 1025),
+              st("C", 1050, null)),
+          t(st("B", 1021, 1025),
+              st("C", 1028, 1030),
+              st("D", 1035, null))
+      ];
+
+      const transfers = {
+          "E": [
+              tf("E", "B", 1)
+          ]};
+
+      const raptor = RaptorQueryFactory.createDepartAfterQuery(trips, transfers, {}, calendars, journeyFactory);
+      const result = raptor.plan("A", "D", new Date("2018-10-16"), 900, ["B"]);
```

Comment:
Missing test cases for multiple not via stations.

Reply:
8c8b9d7 added multiple not via test